syntax = "proto3";
package tinyim;

import "common/messages.proto";
// import "access/access.proto";

option cc_generic_services = true;



service DbproxyService {
    rpc AuthAndSaveSession(SigninData) returns (Pong);
    rpc ClearSession(UserId) returns (Pong);
    // rpc ChangeSession(UserId) returns (Pong);

    rpc SavePrivateMsg(NewPrivateMsg) returns (Reply);
    rpc SaveGroupMsg(NewGroupMsg) returns (Reply);
    // rpc SaveGroupMsg() returns (Reply);
    // rpc GetMsg(UserId) returns (Msg);
    rpc GetGroupMember(GroupId) returns (UserIds);
}